### PR TITLE
feat: created seed, a new script, fixed some problems with seeds

### PR DESCRIPTION
**Pull request by Stanislav-Vyhnan**

### CODE REVIEW

File: src/controllers/unitofmeasurings.js [hunk 1]
```diff
@@ -1,51 +1,51 @@
...
 };
 
 const getUnitOfMeasuringsById = async (req, res) => {
-    const unitOfMeasuring = await UnitOfMeasurings.findAll({
-      where: {
-        id: req.params.id,
-      },
-    });
-    res.json(unitOfMeasuring[0]);
+  const unitOfMeasuring = await UnitOfMeasurings.findAll({
```
Comment (VadymKharyton) [inline src/controllers/unitofmeasurings.js:14]: Please replace with `findOne` method call

File: package.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "start": "cross-env NODE_ENV=production node src/index.js",
     "start:dev": "cross-env NODE_ENV=development nodemon src/index.js",
+    "run-seeds": "cd src/ & npx sequelize-cli db:migrate:undo:all && npx sequelize-cli db:migrate && npx sequelize-cli db:seed:all",
```
Comment (VadymKharyton) [inline package.json:9]: That's what I asked, great, thanks. 
But please rename this script into `db-migration`

File: src/controllers/unitofmeasurings.js (hunk 1 above)
Comment (Stanislav-Vyhnan) [reply src/controllers/unitofmeasurings.js]: Replaced ✔️

File: package.json (hunk 2 above)
Comment (Stanislav-Vyhnan) [reply package.json]: Renamed ✔️

**Review by VadymKharyton — APPROVED** [review]

**Pull request merged**